client/src/components/Countdown.tsx

```diff
@@ -20,9 +20,11 @@ const defaultCountdownRenderer = (
 interface CountdownProps {
   date: Moment
   onComplete?: ReactCountdownProps['onComplete']
+  overTime?: boolean
```

Comment:
We should keep the casing that is used in ReactCountdown. This does not mean "over time" but "overtime" (one word). I don't think it's very descriptive but let's at least be consistent.